Sources/DashX/DashXAppDelegate.swift

```diff
@@ -74,11 +75,14 @@ open class DashXAppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificat
         notificationContent.title = dashxData.title
         notificationContent.body = dashxData.body
         notificationContent.userInfo = userInfo
-
-        let request = UNNotificationRequest(identifier: dashxData.id, content: notificationContent, trigger: nil)
-
-        UNUserNotificationCenter.current().add(request)
-
+        
+        if let imagePath = dashxData.image,
+           let imageURL = URL(string: imagePath) {
+            dashXClient.createNotificationWithImage(id: dashxData.id, imageURL: imageURL, notificationContent: notificationContent)
```

Comment:
Should `notificationContent` parameter be `content`? 
Similar to `createNotification(id: dashxData.id, content: notificationContent)`

Reply:
Done